refactored classes api

RobertShaw1 opened the pull request:
We have all the api  routes we need for '/classes'.  I have tested them with POSTMAN and they work as expected.  Feel free to double check.  

After this branch is reviewed, approved, and pull into master we can delete this branch.

RobertShaw1 committed:
refactored classes api

ghanley reviewed (commented):
I dont think we should remove the ID route

Commit:
Update classes.js

re-adding the route to get a single class since the group preference seems to be that we keep it.
